11-payment-encoding.md

```diff
@@ -293,6 +293,7 @@ on the amount sent.
 
 A writer:
   - if `payment_secret` feature is set:
+    - MUST set the `var_onion_optin` feature.
```

Comment:
very nit: we could remove that requirement now that Bolt 9 already states this.

Reply:
@t-bast turns out we can simplify this section quite a bit by referring back to the implicit requirements in BOLT 9. lmk what you think of the latest version